web/app.js

```diff
@@ -419,10 +419,14 @@ var PDFViewerApplication = {
     var support = !!(doc.requestFullscreen || doc.mozRequestFullScreen ||
                      doc.webkitRequestFullScreen || doc.msRequestFullscreen);
 
+//#if MOZCENTRAL
```

Comment:
yes, doc.requestFullscreen will present anyway, also `document.fullscreenEnabled === true` will make sure it's enabled and at the same moment will check if API is present (`=== true`) is a key :)